R/update-version.R

```diff
@@ -24,6 +16,21 @@ update_version_impl <- function(which) {
   desc$write()
 }
 
+update_version_helper <- function(which) {
+  desc <- desc::desc(file = "DESCRIPTION")
+
+  if (desc$has_fields("Date")) {
+    desc$set("Date", Sys.Date())
+  }
+
+  # https://github.com/r-lib/desc/issues/93
+  suppressMessages(desc$bump_version(which))
+
+  new_version <- desc$get_version()
+
+  return(new_version)
```

Comment:
Should this function return the modified `desc` object, and be used in `update_version_impl()` ?